docs/en/00_Getting_Started/02_Composer.md

```diff
@@ -56,48 +64,57 @@ see [Using development versions](#using-development-versions).
 
 Composer isn't only used to download SilverStripe CMS, it can also be used to manage all SilverStripe modules.  Installing a module can be done with the following command:
 
-	composer require "silverstripe/forum:*"
+```
+composer require "silverstripe/forum:*"
+```
+
+This will install the forum module in the latest compatible version. You can also omit the `:*` if you simply want the latest compatible version of the module.
```

Comment:
good point. For the moment * seems to be a time saver, but later, when you want to update your site/app it's installing the latest stable compatible version - which might have breaking changes and unintended side effects to your existing app. This is a problem if your code relies on the installed module.